fixbaserom.py

```diff
@@ -1,37 +1,48 @@
-import os.path
 from os import path
 import sys
 import struct
 import hashlib
 
 
+def get_str_hash(byte_array):
+    return str(hashlib.md5(byte_array).hexdigest())
+
+
+# If the baserom exists and is correct, we don't need to change anything
+if path.exists("baserom.z64"):
+    with open("baserom.z64", mode="rb") as file:
+        fileContent = bytearray(file.read())
+        if get_str_hash(fileContent) == "f0b7f35375f9cc8ca1b2d59d78e35405":
+            print("Found valid baserom - exiting early")
+            sys.exit(0)
+
 # Determine if we have a ROM file
 romFileName = ""
-if (path.exists("baserom_original.z64")):
+if path.exists("baserom_original.z64"):
     romFileName = "baserom_original.z64"
-elif (path.exists("baserom_original.n64")):
+elif path.exists("baserom_original.n64"):
     romFileName = "baserom_original.n64"
```

Comment:
still think we should go with input.z64 to clear up the confusion about the meaning of original. readme and error message below would need updating as well

Reply:
oh right. one sec

Reply:
input is more confusing than original imo.

Reply:
As far as I can tell, the readme doesn't mention baserom_original - did you mean the wiki?